File: frontend/src/domain/enums/UserType.ts
export enum UserType {
  GlobalAdmin = 1,
  OrganizationAdmin = 2,
  OrganizationUser = 3,
  CommissionUser = 4,
  InterestedParty = 5,
}

export const UserTypeLabels: Record<UserType, string> = {
  [UserType.GlobalAdmin]: 'Global Admin',
  [UserType.OrganizationAdmin]: 'Organization Admin',
  [UserType.OrganizationUser]: 'Organization User',
  [UserType.CommissionUser]: 'Commission User',
  [UserType.InterestedParty]: 'Interested Party',
};

File: backend/FoodEval.Domain/Enums/UserType.cs
namespace FoodEval.Domain.Enums;

public enum UserType
{
    GlobalAdmin = 1,        // SuperAdmin
    OrganizationAdmin = 2,   // Admin of an Organization
    OrganizationUser = 3,   // Normal user inside an Organization
    CommissionUser = 4,     // Commission-related users
    InterestedParty = 5     // Email notification only
}
